### DISCUSSION TITLE
Can I have more than one model in one app?

### DISCUSSION THREAD

**Discussion by paulwrightwork** [Project-MONAI/MONAILabel#654] [Q&A] [answered]
I might want to compare, for example, the output of deepedit and deepgrow on the same set of images. I notice there are dropdowns to select which model to train. That implies my app can include more than one model. To what extent does MONAIlabel support multiple models (trainers / inferrers/ ?) and how does one go about putting them in ones app?

**Comment by SachidanandAlle** [comment, marked as answer]
Yes.. you can have multiple models.. and app developer can decide what to train/infer etc..

Example (it has 2D and 3D models)
https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/deepgrow/main.py#L67-L124
